tempus_fugit_minecraft/model.py

```diff
@@ -134,15 +134,12 @@ def exposed(self, position: tuple) -> bool:
         return False
 
     def add_block(self, position: tuple, texture: list, immediate=True) -> None:
-        """Add a block with the given `texture` and `position` to the world.
+        """!
```

Comment:
What does this mean? (!)

Reply:
The exclamation mark or `!` is what to tell `DOXYGEN` how to treat the comments and added them to the documentation.